includes/polling/os/awplus.inc.php

```diff
@@ -9,7 +9,16 @@
 $serial = snmp_getnext($device, "rscBoardSerialNumber", "-OQv", "AT-RESOURCE-MIB");
 
 // SBx8100 platform has line cards show up first in "rscBoardName" above.
+//Instead use sysObjectID.0
 if (strpos($hardware, 'SBx81') !== false) {
     $hardware = snmp_get($device, "sysObjectID.0", "-OQvs", "SNMPv2-MIB:AT-PRODUCT-MIB");
-    $hardware = str_replace('at', '', $hardware);
+    $hardware = str_replace('at', 'AT-', $hardware);
```

Comment:
Is this not risking matching at anywhere in the hardware string? Would `'/^at/'` not be better?

Reply:
Yeah good point.

Reply:
I've spent a while on this and can't get something that will match.

'/^at/' doesn't seem to work for atSBx8106 or atSBx8112.

Unless you can give another suggestion, I really don't know what to try :-(

These are the only two options $hardware can be, and given that we only go down this path if SBx81 is true I guess this shouldn't be too concerning.